com/bl/ttt/tictactoe.py

```diff
@@ -53,6 +56,10 @@ def _is_any_diagonal_complete(self, player_id):
                 self.cells[6] == self.cells[4] == self.cells[2] == player_id:
             return True
 
+    def is_draw(self):
+        if ' ' not in self.cells:
```

Comment:
how about 
`return ' ' not in self.cells`

Reply:
Done